content/tutorials/configuration.md

```diff
@@ -113,6 +114,42 @@ tls:
 Optionally, also configure a TLS proxy in this section. 
 See the separate [TLS Proxy page]({{< relref "concepts/tls-proxy" >}}) for more information.
 
+#### API Configuration
+
+By default, KES requires a valid TLS certificate for all API calls.
+You can change this behavior for specific API endpoints to allow requests to the endpoints without a certificate.
+
+{{< admonition type="caution" >}}
+The default behavior of the KES API endpoints should suffice in most every situation.
+Only customize endpoints for specific need
+{{< /admonition >}}
```

Comment:
Slightly shorter alternative?
> The default behavior of the KES API endpoints should be suitable for most situations. Only customize endpoints for a specific need.

Reply:
Not much shorter, but probably clearer. Going with your suggestion.